### ISSUE TITLE
gdma and fchk

### ISSUE THREAD

**Issue by loriab** [psi4/psi4#1523]
No merge goes unpunished ...

Something of #1475 and the gdma interface aren't playing nicely.

I'll look at it, but posting here for general notification and in case anyone takes it up while I do some meetings.

```
(p4dev37) psilocaluser@bash:psinet:/home/psilocaluser/gits/hrw-quaternary/objdir37: (master) stage/bin/psi4 ../tests/gdma/gdma1/input.dat 

RHF density not found
Job abandoned
```

**Comment by dgasmith** [comment]
Closed by #1523.

**Comment by dgasmith** [comment]
I apparently cannot count.
